docs/bot/configuration.md

```diff
@@ -36,6 +37,7 @@ These are the keys you can specify:
       "link": "[<%= symbol %>](<%= url %> \"<%= description %>\"),"
     }
   },
+  "skipCi": 'true',
```

Comment:
Looks good so far.
There's just a minor error on line 40 (since JSON doesn't support `'`).

Reply:
Whoops.
I didn't known that.
Fixing ASAP.